SpywareFilter/sections/specific.txt

```diff
@@ -50,6 +50,7 @@ semprotyuk.com##div[style^="margin-bottom:150px;"][style*="padding-right:2em;"]
 ||api.nvidia.partners/telemetry/
 ||dax.yle.fi/yya/event
 ||nature.com/platform/track/
+||cdn.shopify.com/*/consent-tracking-api/*/consent-tracking-api.js
```

Comment:
It can break the store. Dangerous.

Reply:
Ok.

Reply:
If I am not mistaken, it's in EasyPrivacy.

Reply:
@al

Reply:
@Alex-302

Reply:
not sure. Better remove, and block requests made by this script

Reply:
I don't see anymore requests. I will remove it.

Reply:
@ale

Reply:
@Alex-302